sbin/build.sh

```diff
@@ -707,8 +707,12 @@ moveFreetypeLib() {
 
   # codesign freetype before it is bundled
   if [ ! -z "${BUILD_CONFIG[MACOSX_CODESIGN_IDENTITY]}" ]; then
-    ENTITLEMENTS="$WORKSPACE/entitlements.plist"
-    codesign --entitlements "$ENTITLEMENTS" --options runtime --timestamp --sign "${BUILD_CONFIG[MACOSX_CODESIGN_IDENTITY]}" "${SOURCE_LIB_NAME}"
+    if touch test && codesign --sign "Developer ID Application: London Jamocha Community CIC" test && rm -rf test; then
```

Comment:
Worth a comment saying what this effectively does.

Reply:
added